docs/website/docs/examples/qdrant_zendesk/code/.dlt/secrets.toml

```diff
@@ -0,0 +1,8 @@
+[destination.qdrant.credentials]
```

Comment:
is this necessary? last time someone pushed a secrets file full of credentials by accidents. this should be removed and IMO never forced to be added. at least use something like `secrets.example.toml` as name